### PR TITLE
added some Turkish cafe/fastfoods

### PR DESCRIPTION
**Pull request by smirnov-e**
Some were requested in https://github.com/osmlab/name-suggestion-index/issues/7706

### CODE REVIEW

**Comment by Cj-Malone** [comment]
We don't have website tags here. Ideally the brand homepage will be on Wikidata, and OSM should have the POI specific website, but NSI holds neither.

**Comment by smirnov-e** [comment]
@Cj-Malone fixed, added one more cafe.

File: data/brands/amenity/cafe.json [hunk 1]
```diff
@@ -2372,6 +2372,47 @@
...
+      "locationSet": {
+        "include": [
+          "tr", "az"
+        ]
+      },
+      "tags": {
+        "amenity": "cafe",
+        "brand": "Shakespeare Coffee & Bistro",
+        "brand:wikidata": "Q119561507",
+        "cuisine": "bistro;coffee_shop;turkish;breakfast;beef;chicken;salad;pasta;pizza;dessert",
```
Comment (LaoshuBaby) [inline data/brands/amenity/cafe.json:2404]: NSI isn't a menu, can you shorten the cuisine list and only focus on major one (or two)?

Also for `drink:*` keys.

Comment (batyrmastyr) [reply data/brands/amenity/cafe.json]: Ok, removed.

**Pull request merged**